### DISCUSSION TITLE
🛡 Dungeons e Domínios ⚔

### DISCUSSION THREAD

**Discussion by ghost** [leogianfagna/IssueTracker#1036] [Ideia]
### Servidor afetado

Henesys

### Qual o propósito da ideia?

Criar uma alternativa de obtenção de pergaminhos mais dinâmica e divertida, aproveitando super encantamentos e evitando o farm Afk de XP.

### Descrição da ideia

**Como dito no patch notes**, "de tempos em tempos, alguns encantamentos serão **desativados**, obrigando os jogadores a trocar por novos em um **ciclo repetitivo**." 

Gostaria de sugerir mais dois sistemas para conseguir os pergaminhos: **Dungeons e Domínios.**

- **Dungeons:** Disponíveis no Skyten, oferecem aventura e desafios para conquistar itens.

- **Domínios:** Diferentes das Dungeons, são mais parecidos com a "Câmara dos Desafios" ou o sistema de "Mob Arena". 
Neles, os jogadores enfrentam ondas de inimigos, e quanto mais longe chegarem, melhores serão as recompensas.
(Da até para fazer um rank disso, quem foi mais longe.)

A ideia é tornar a coleta de pergaminhos mais interessante, incentivar outros usos dos super encantamentos além de guerras e PvP, nesse caso ao PvE, e ao mesmo tempo, desestimular o farm Afk de XP com intuito de obter pergaminhos.

Se a ideia for aprovada, seria interessante criar um Domínio Divino para conseguir pergaminhos divinos. 

Acredito que seja uma alternativa que forme guildas, times e clans a trabalharem juntos

### Referências

- **Dungeons:** Inspiradas no Skyten.

- **Domínios:** Genshin Impact e Mob Arena

**Comment by leogianfagna** [comment]
Não fica muito parecido com mob arena e skyten?

**Reply by ghost** [reply]
A meu ver, vai ser diferente pela presença dos super encantamentos, dos pets e de tudo que o Henesys oferece. 
Da para fazer missões de npc, modificadores... bem a visão de rpg que foi informada no patch notes.
